recipes/pixman/all/conanfile.py

```diff
@@ -17,40 +16,49 @@ class PixmanConan(ConanFile):
     _autotools = None
 
     @property
-    def _includedir(self):
-        return os.path.join("include", "pixman-1")
+    def _source_subfolder(self):
+        return "source_subfolder"
 
     @property
-    def _folder(self):
-        return "{}-{}".format(self.name, self.version)
+    def _includedir(self):
+        return os.path.join("include", "pixman-1")
 
     def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            del self.options.fPIC
         del self.settings.compiler.libcxx
         del self.settings.compiler.cppstd
-        if self.settings.os == 'Windows':
-            del self.options.fPIC
+        if self.settings.os == "Windows" and self.options.shared:
+            raise ConanInvalidConfiguration("pixman can only built as static library for Windows")
```

Comment:
Grammar nazi, to your service :smile: 
```suggestion
            raise ConanInvalidConfiguration("pixman can only be built as a static library on Windows")"
```